src/Bicep.Core/Analyzers/Linter/Rules/NoUnusedParametersRule.cs

```diff
@@ -39,5 +38,10 @@ override public IEnumerable<IDiagnostic> AnalyzeInternal(SemanticModel model)
 
             return unreferencedParams.Select(param => CreateRemoveUnusedDiagnosticForSpan(param.Name, param.NameSyntax, param.DeclaringSyntax, model.SourceFile.LineStarts, model.SourceFile.ProgramSyntax));
         }
+
+        override protected string GetCodeFixDescription()
+        {
+            return "Remove unused parameter";
```

Comment:
nit: Could you please make this "Remove unused parameter '<paramname>'"?  It's possible to select a region of code, in which case they can get multiple fixes that look alike but apply to different params/vars.

If you're getting sick of this PR and would rather get it in now, that's fine, I could do it myself later.